src/pipeline/prep_line_list.jl

```diff
@@ -2,7 +2,7 @@
 """ prepare_line_list( linelist_fn, spectra, pipeline; Δv_to_avoid_tellurics, v_center_to_avoid_tellurics )
 linelist_fn is the full path to the line list file
 """
```

Comment:
Let's mention the airToVac flag in the docstring, including its default value and that it is only pass along if the linelist_fn matches r"espresso\.mas$" or r"VALD".

Reply:
Also convention is avoid capital letters for local variables like this.  Maybe rename "convert_air_to_vacuumb"?